plugins/gtav/structs.h

```diff
@@ -0,0 +1,227 @@
+// Copyright 2022 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#ifndef GTAV_STRUCTS
+#define GTAV_STRUCTS
+
+#include <cstdint>
+
+// Reference: https://github.com/Yimura/GTAV-Classes
```

Comment:
Technically that repo does not contain a LICENSE file and as such we are not allowed to use any of that.

@Yimura mind adding a LICENSE file to your repo? Preferably something that is compatible with BSD-3-Clause

Reply:
Sure I don't mind, is MIT alright with you? It's what I usually pick.